arch/arm/src/cxd56xx/cxd56_spi.c

```diff
@@ -77,7 +77,6 @@ struct cxd56_spidev_s
   uint8_t          port;        /* Port number */
   int              initialized; /* Initialized flag */
 #ifdef CONFIG_CXD56_DMAC
-  bool             dmaenable;   /* Use DMA or not */
   DMA_HANDLE       rxdmach;     /* RX DMA channel handle */
```

Comment:
Yes, because txdmach and rxdmach doesn't initialize at runtime anymore, so dmaenable which is used to identify the initialization result doesn't need too.